### PR TITLE
Twblack contributing patch

### PR DESCRIPTION
**Pull request by twblack88**
CDIP guide, readme edits, and the image for the CDIP are in this PR. 

We may have to adjust file paths for gitbooks or create a folder but I think this covers the process we've outlined so far.

### CODE REVIEW

File: contributing/CDIP explainer.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# CDIP Explainer
+
+## What is a CDIP
+
+CDIP stands for Community Development Initiative Proposal. The CDIP is the scoping and framing tool for for anyone interested in contributing to the MakerDAO community initiatives or resources. CDIPs are used to propose new initiatives or expand the scope of [existing initiatives and resources](https://community-development.makerdao.com/). A proposal should combine the contributor’s ideas, skills, and abilities with our shared goal of improving the resources available to the MakerDAO community.
```
Comment (ghettodev) [inline contributing/CDIP explainer.md]: links to the gitbooks should be relative

File: contributing/CDIP explainer.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+# CDIP Explainer
+
+## What is a CDIP
```
Comment (ghettodev) [inline contributing/CDIP explainer.md]: Why do none of these questions have a question mark at the end?

File: contributing/CDIP explainer.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+### What are the benefits of a CDIP
+
+The CDIP provides a standard contributing template for Maker’s community. The format provides a few benefits: collect and organize relevant information in a standardized format to make ideas easier to understand, add an efficient and transparent evaluation layer to contribution proposals, simplify the ability to archive ideas and search details about them later, and ultimately serve as a tool to streamline how we review and approve work as a community. CDIPs scale the ability to execute the ideas of the community.
+
+### How is a CDIP started
+
+Use this [CDIP template doc](http://bit.ly/Comm_Dev_CDIP) to start the process. To submit a CDIP, simply make a copy of the template, fill out the first page (step 1), and post a link to the doc in our group #[Community Development](https://chat.makerdao.com/channel/community-development) rocketchat. A Guide will reach out to help the CDIP owner through the next steps.
```
Comment (ghettodev) [inline contributing/CDIP explainer.md]: Lets avoid the use of url shorteners. This also links to a personal account. 
lets not use the name 'rocketchat/, please just call it our chat.

File: contributing/CDIP explainer.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+### How does a CDIP work
+
+Below is a visual representation of our current process. Contributor steps are highlighted in green, approving group steps are highlighted in purple, and overlapping steps are highlighted in orange. A community member who acts as a Guide assists new CDIP owners in refining and bringing their CDIP towards completion by coordinating on the orange steps.
+
+![How To CDIP](https://github.com/makerdao/community/blob/Twblack-Contributing-Patch/contributing/CDIP%20Contributor%20highlights.jpg)
```
Comment (ghettodev) [inline contributing/CDIP explainer.md]: this links to your repo

File: contributing/CDIP explainer.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+### How does a CDIP Owner ask for resources
+
+Each CDIP includes a section on costs and resources. As part of the approval process, the owner of the CDIP presents the idea along with the required resources. Resources can include, but not limited to:
+
+People support (developer, designer, editor, writer, videographer, etc)
```
Comment (ghettodev) [inline contributing/CDIP explainer.md]: are these list items?

File: contributing/CDIP explainer.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+Foundation resources support (brand assets, official product screenshots, etc)
+
+Once the CDIP is approved, these resources are granted. As a general rule of thumb, CDIPs cover costs under 5k DAI. The specific details vary case by case.
+
+For general resources, if what’s needed is not in the [contributing section](https://community-development.makerdao.com/contributing) of the community development portal, come onto the [#community-development](https://chat.makerdao.com/channel/community-development) channel on our chat and ask.
+What happens if a CDIP needs more resources than originally proposed?
```
Comment (ghettodev) [inline contributing/CDIP explainer.md]: is this a new heading?

File: contributing/README.md [hunk 7]
```diff
@@ -26,11 +26,12 @@ Community members have numerous options when contributing to resources in the Ma
 
 ## How do I get involved?
 
-Get familiar with the [style guide](./style-guide.md). 
+Get familiar with the [style guide](./style-guide.md). Learn more about [CDIP's](https://github.com/makerdao/community/blob/Twblack-Contributing-Patch/contributing/CDIP%20explainer.md)
```
Comment (ghettodev) [inline contributing/README.md]: this links to your repo, use relative links please

File: contributing/README.md [hunk 8]
```diff
@@ -26,11 +26,12 @@ Community members have numerous options when contributing to resources in the Ma
...
 
 Then, decide where you want to contribute:
 
 - [Translate MakerDAO resources](/translations/README.md)
 - [Improve Meetup guides](/meetups/README.md)
 - [Curate Awesome-MakerDAO](https://github.com/makerdao/awesome-makerdao)
 - [MakerDAO: Help Wanted](https://forum.makerdao.com/c/comm-dev/help-wanted)
+- [Submit your own CDIP](http://bit.ly/Comm_Dev_CDIP)
```
Comment (ghettodev) [inline contributing/README.md]: no url shorteners

**Comment by twblack88** [comment]
Added two commits two weeks ago. Should be good to go! Let me know if relative linking looks correct.

**Pull request merged**